tpl/tplimpl/template_embedded.go

```diff
@@ -84,7 +84,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
       <pubDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</pubDate>
       {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
       <guid>{{ .Permalink }}</guid>
-      <description>{{ .Content | html }}</description>
+      <description>{{ .Summary | html }}</description>
```

Comment:
@DarwinJS Sure thing. I think the point of confusion is the terms we are using, and I apologize for any confusion.

> **tl;dr
>  1. Hugo `.Summary` == "lead"
>  2. Hugo `.Description` == user-defined summary of content (e.g., synopsis or meta description)
>  3. `.Content` is part of Atom and not RSS

**Explanation**

The `.Summary` in Hugo is really a ["lead" in publishing](https://en.wikipedia.org/wiki/Lead_paragraph). So, a first paragraph, or a specific word count, *could* be a summary of the article intended to draw the reader in, but I think this assumes too much from authors, at least with respect to creating a Rich Site Summary. Of course, the feature is pretty darn handy, especially with the ability for users to define their own via `<!--more-->`. The summaries Hugo creates automatically are really *excerpts* or *extracts*. This point has been called out more specifically in the [new Hugo docs](https://hugodocs.info/content-management/summaries/#user-defined-manual-summary-splitting) in hopes of making this clearer. (Please provide feedback if it doesn't! :smile:) 

A *description* is a more focused and deliberate paragraph of text that does in fact summarize the full content of the page, although&mdash;I'm guessing&mdash;the way it was originally intended in Hugo was in the case of meta descriptions (usually <160 characters)...which also makes sense.  

As far as the RSS `<description>` tag containing the full-length `.Content`, this is a case of confusing RSS with [Atom](https://en.wikipedia.org/wiki/RSS#RSS_compared_to_Atom), which allows for a *optional* summary *or* full content, whereas RSS requires it. Here is an excellent [resource from Harvard on the RSS spec](http://cyber.harvard.edu/rss/rss.html#hrelementsOfLtitemgt). NB, `<description>` in RSS can either mean "channel description" or "item synopsis." 

So that's my reasoning behind the `.Description => .Summary` since it follows a model of `deliberate usage => sane default`. 

In case you are asking about the `default` syntax specifically, and excuse me if you know this, but I love this function and feel like it's underused: you can see the [docs on it here](https://hugodocs.info/functions/default-function/) as well.

HTH.